Week7/SOLUTION WEEK-7/WEEK-7 ASSIGNMENT-checkpoint.ipynb

```diff
@@ -0,0 +1,2957 @@
+       "      <td>700</td>\n",
+       "      <td>750</td>\n",
+       "      <td>800</td>\n",
+       "      <td>730</td>\n",
+       "      <td>770</td>\n",
+       "      <td>790</td>\n",
+       "      <td>4540</td>\n",
+       "      <td>756.666667</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>12.0</th>\n",
+       "      <td>P13</td>\n",
+       "      <td>NOTEBOOK</td>\n",
+       "      <td>700</td>\n",
+       "      <td>710</td>\n",
+       "      <td>720</td>\n",
+       "      <td>730</td>\n",
+       "      <td>730</td>\n",
+       "      <td>750</td>\n",
+       "      <td>3610</td>\n",
+       "      <td>601.670000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>13.0</th>\n",
+       "      <td>P14</td>\n",
+       "      <td>CALC</td>\n",
+       "      <td>70</td>\n",
+       "      <td>75</td>\n",
+       "      <td>72</td>\n",
+       "      <td>73</td>\n",
+       "      <td>73</td>\n",
+       "      <td>75</td>\n",
+       "      <td>438</td>\n",
+       "      <td>73.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2.5</th>\n",
+       "      <td>P15</td>\n",
+       "      <td>PENCIL</td>\n",
+       "      <td>500</td>\n",
+       "      <td>510</td>\n",
+       "      <td>520</td>\n",
+       "      <td>530</td>\n",
+       "      <td>560</td>\n",
+       "      <td>550</td>\n",
+       "      <td>3170</td>\n",
+       "      <td>528.330000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3.5</th>\n",
+       "      <td>P16</td>\n",
+       "      <td>FOLDER</td>\n",
+       "      <td>400</td>\n",
+       "      <td>410</td>\n",
+       "      <td>420</td>\n",
+       "      <td>430</td>\n",
+       "      <td>460</td>\n",
+       "      <td>450</td>\n",
+       "      <td>2570</td>\n",
+       "      <td>428.330000</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     PRODUCT_NO PRODUCT_NAME  JANUARY  FEBRUARY  MARCH  APRIL  MAY  JUNE  \\\n",
+       "11.0        P12       FOLDER      700       750    800    730  770   790   \n",
+       "12.0        P13     NOTEBOOK      700       710    720    730  730   750   \n",
+       "13.0        P14         CALC       70        75     72     73   73    75   \n",
+       "2.5         P15       PENCIL      500       510    520    530  560   550   \n",
+       "3.5         P16       FOLDER      400       410    420    430  460   450   \n",
+       "\n",
+       "      TOTAL_SALES  AVERAGE_SALES  \n",
+       "11.0         4540     756.666667  \n",
+       "12.0         3610     601.670000  \n",
+       "13.0          438      73.000000  \n",
+       "2.5          3170     528.330000  \n",
+       "3.5          2570     428.330000  "
+      ]
+     },
+     "execution_count": 40,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df.tail()"
```

Comment:
is it possible to get last 5 records using head and if yes how?

Reply:
Basically yes... 
df.iloc[::-1].head()..